src/Operators/finitedifference.jl

```diff
@@ -1787,7 +1989,182 @@ Base.@propagate_inbounds function stencil_right_boundary(
     return Geometry.Contravariant3Vector(zero(eltype(eltype(A_field))))
 end
 
+"""
```

Comment:
Can this docstring also be attached to the relevant operator?

Reply:
It's a bit duplicate, but I added a short stub so that users can navigate to the general documentation.